<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOESDemo</class>
 <widget class="QMainWindow" name="IOESDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IOESDemo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,20,0,1,1,1">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>8</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>服务URL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combxSericeType">
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="edtURL">
          <property name="text">
           <string>http://192.168.1.222:9098/images/recog</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLineEdit" name="edtImagePath">
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBraws">
            <property name="text">
             <string>选择图片目录</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>请求模式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combxModel"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>单组图片：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combxBunchSize"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listImages">
          <property name="autoScroll">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="edtX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="statusTip">
             <string extracomment="x"/>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhPreferNumbers</set>
            </property>
            <property name="text">
             <string>10, 10, 100, 100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMarkRect">
            <property name="text">
             <string>标注</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>图片总数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTotalImageNum">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>已识别数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblParsedImageNumber">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>耗时(ms)：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTimeCost">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnStartTask">
            <property name="text">
             <string>提交任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStopTask">
            <property name="text">
             <string>终止任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDumpResult">
            <property name="text">
             <string>导出结果</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,0,3">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="gvPreview"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="cbxPerson">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>行人</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxFace">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 0, 255);</string>
            </property>
            <property name="text">
             <string>人脸</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxHead">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 0);</string>
            </property>
            <property name="text">
             <string>头部</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBody">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 255, 255);</string>
            </property>
            <property name="text">
             <string>身体</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxCar">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 255, 0);</string>
            </property>
            <property name="text">
             <string>机动车</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxBike">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 85, 255);</string>
            </property>
            <property name="text">
             <string>非机动车</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
          <item>
           <widget class="QTextBrowser" name="mtxtResponse"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="mtxtParseResult"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1424</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="menuConfigDumpPicture"/>
    <addaction name="menuConfigSelfCheck"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuCfgFull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full</string>
   </property>
  </action>
  <action name="menuCfgPBC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Person+Bike+Car</string>
   </property>
  </action>
  <action name="menuCfgFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Face</string>
   </property>
  </action>
  <action name="menuCfgPB">
   <property name="text">
    <string>Person+Bike</string>
   </property>
  </action>
  <action name="menuCfgCar">
   <property name="text">
    <string>Car</string>
   </property>
  </action>
  <action name="menuCfgBunch1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="menuCfgBunch5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="menuCfgBunch10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="menuCfgThread1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="menuCfgThread4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="menuCfgThread8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="menuCfgThread12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="menuConfigDumpPicture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DumpWithPicture</string>
   </property>
  </action>
  <action name="menuConfigSelfCheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SelfCheck</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
